### DISCUSSION TITLE
What does deployment look like?

### DISCUSSION THREAD

**Discussion by mnishizawa** [lunatic-solutions/lunatic#130] [Q&A]
Does lunatic require a client-side executable or does it run inside the browser?  If I wanted to build this into a commercial application is this something that can be deployed without the user having to perform extra steps?

**Comment by bkolobara** [comment]
There are two parts to lunatic:

1. The lunatic-runtime (this repository).
2. WebAssembly applications that use the functionality that the lunatic-runtime provides.

The lunatic-runtime works on Windows/Linux/macOs (x64 & aarch64) and is required to run .wasm files containing the actual application. Of course, you can build web apps with lunatic too. In this case the user just needs to point their browser to your server, and your server will run the .wasm app using the lunatic-runtime.
